CMakeLists.txt

```diff
@@ -270,6 +270,29 @@ cxx_feature_check(STEADY_CLOCK)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
+# If successful, the following variables will be defined:
+# HAVE_LIBPFM.
+# Set BENCHMARK_ENABLE_LIBPFM to 0 to disable, regardless of libpfm presence.
+include(CheckIncludeFile)
+include(CheckLibraryExists)
+ENABLE_LANGUAGE(C)
+
+if (BENCHMARK_ENABLE_LIBPFM)
+  check_library_exists(libpfm.a pfm_initialize "" HAVE_LIBPFM_INITIALIZE)
```

Comment:
This could be encapsulated as `FindPFM.cmake` module and used as `find_package(PFM)`. I can do this conversion if desired.

Reply:
thanks! I can take a stab at it once build bots green up, as a last step.

Reply:
done